source/include/VirtualInstrument.h

```diff
@@ -246,17 +248,17 @@ class ClumpParseState
     InstructionAllocator* _cia;
     
     Int32           _argCount;
-    void*           _argPointers[kMaxArguments];
-    TypeRef         _argTypes[kMaxArguments];
-    
-    Int32           _argPatches[kMaxArguments];     // Arguments that need patching
+    std::vector<void*> _argPointers;
```

Comment:
I'm OK on leaving this code for the pull request. If we decide we want to do something else, we can remove it later. I'm more concerned about the tests not passing on windows.